model-optimizer/mo/utils/versions_checker.py

```diff
@@ -44,34 +44,74 @@ def check_python_version():
         return 1
 
 
-def parse_versions_list(required_fw_versions, version_list):
+def parse_versions_list(required_fw_versions, version_list, env_params=dict()):
```

Comment:
Remove '=dict()'. Read the comment in the code below!

Reply:
Fixed.